### DISCUSSION TITLE
splash page and sidebar

### DISCUSSION THREAD

**Discussion by jphconstantin** [mmistakes/minimal-mistakes#2667] [Q&A] [answered]
Why a page with layout: splash doesn't displayed a sidebar ?

```
layout: archive
sidebar:
  nav: "my_nav"
```
the sidebar is displayed

```
layout: splash
sidebar:
  nav: "my_nav"
```
the sidebar isn't displayed
"my_nav" defined in navigation.yml

**Comment by mmistakes** [comment, marked as answer]
The splash layout was designed not to have a sidebar. Use `single` or one the archive layouts if needed.
